docs/introduction/overview.md

```diff
@@ -43,6 +41,14 @@ Phoenix is made up of a number of distinct parts, each with its own purpose and
     - underlies the channel layer and allows clients to subscribe to *topics*
     - abstracts the underlying pubsub adapter for third-party pubsub integration
 
+## Phoenix Layers
+
+We just covered the internal parts that make up Phoenix, but its important to remember Phoenix itself is actually the top layer of a multi-layer system designed to be modular and flexible. The other layers include Cowboy, Plug, and Ecto.
+
+### Cowboy
+
+We won't be covering the Erlang HTTP server, Cowboy, in these guides other than to note that Cowboy acts as the foundation for both Phoenix and Plug.
```

Comment:
Could we change this to:

 > By default, the web server used by Phoenix (and Plug) is Cowboy. It is uncommon to interface with Cowboy directly when using Phoenix. If you do require using Cowboy directly, please refer to [the Cowboy documentation](https://ninenines.eu/docs/en/cowboy/1.0/guide/)